packages/circuits/crates/libs/greco/src/lib.nr

```diff
@@ -163,7 +163,7 @@ impl<let N: u32, let L: u32> Params<N, L> {
 /// * `k1` - Scaled message polynomial
 /// * `r1is`, `r2is` - Randomness polynomials for each i-th CRT basis.
 /// * `p1is`, `p2is` - Randomness polynomials for each i-th CRT basis.
-pub struct Greco<let N: u32, let L: u32> {
+pub struct Greco<let N: u32, let L: u32, let BIT_PK: u32, let BIT_U: u32, let BIT_E: u32, let BIT_K: u32, let BIT_R1: u32, let BIT_R2: u32, let BIT_P1: u32, let BIT_P2: u32> {
```

Comment:
can you add some comments on the `# Generic Parameters` section above for `BIT_PK`, `BIT_U` etc.